### PR TITLE
create Actionnable expandible table & Expandible errors table

### PR DESCRIPTION
**Pull request by youssefezzahi96**
[ticket](https://trello.com/c/YbO0HtBE/1188-composant-inspect-post-progress-100) 
Actionnable errors table is a component that would be used on bulk tables. and is built using errors table, and expandible table already created.
Expandible errors table is a component that would be used to build the other bulk tables. and is build using errors table already created.
Purpose of the PR:

- create actionnable expandible errors table.
- create expandible errors table.
- actionnable expandible errors table  & expandible errors table should be RGAA compliant
- update wizardContents to support expandible errors table
- update listItems to support actionnable expandible errors table
- add actionnable expandible errors table  & expandible errors table to brandUpdate
- create bulk inspect fixtures
- create bulk dashboard fixtures
- actionnable expandible errors table &  expandible errors table test coverage 100%

**Expandible errors table  :**
Pasted Graphic 32

Pasted Graphic 33

Pasted Graphic 34

brandUpdate Template bulk inspect: 
image

**Actionnable expandible errors table  :**
image
image
image
image

**Action menu:**
image

**sticky columns when scrolling horizontally:**
image
image

dashboard / saved : 
image

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
 import React, {useCallback} from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
 import ButtonLinkIconOnly from '../../atom/button-link-icon-only';
 import ButtonMenu from '../../atom/button-menu';
+// eslint-disable-next-line css-modules/no-unused-class
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.js:6]: pk?

File: packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.js [hunk 2]
```diff
@@ -14,7 +24,12 @@ const BulletPointMenuButton = props => {
   };
 
   const menu = (
-    <div className={style.bulletPointMenu} data-name="menu-wrapper" aria-label={menuAriaLabel}>
+    <div
+      // className={menuClassName ? style[menuClassName] : style.bulletPointMenu}
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.js:28]: ton commentaire

File: packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+  const headerRow = columns.map((column, cIndex) => {
+    const {title} = column;
+    return (
+      <th
+        className={classnames(style[`header-${cIndex}`], style.header)}
+        key={`${title}-${cIndex}`}
+        role="columnheader"
+      >
+        <div>{title}</div>
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx:65]: pourquoi tu ajoutes une `div`?

File: packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      </th>
+    );
+  });
+
+  headerRow.push(
+    <th className={style[`header-last`]} key="action-header">
+      <div> {''}</div>
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx:72]: idem

File: packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+  );
+  const headerView = [...headerRow];
+
+  const bodyView = rows.map((row, index) => {
+    const {fields = []} = row;
+    const bodyRow = fields.map((field, fIndex) => {
+      return (
+        <td className={classnames(style[`col-${fIndex}`], style.col)} key={`${field}-${fIndex}`}>
+          {
+            // eslint-disable-next-line no-nested-ternary
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx:83]: on évite de désactiver cette regle

File: packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+            fIndex === 0 && isString(field) ? (
+              <div>
+                <ButtonLinkIconOnly
+                  // eslint-disable-next-line react/jsx-no-bind
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-actionable-table/index.tsx:87]: idem

File: packages/@coorpacademy-components/src/molecule/expandible-actionable-table/style.css [hunk 7]
```diff
@@ -0,0 +1,98 @@
+@value colors: "../../variables/colors.css";
+@value xtraLightGrey from colors;
+@value cm_grey_400 from colors;
+@value cm_grey_700 from colors;
+@value cm_negative_red_200 from colors;
+@value light from colors;
+
+.wrapper {
+  /* overflow-x: auto;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-actionable-table/style.css:9]: le commentaire

File: packages/@coorpacademy-components/src/molecule/expandible-errors-table/index.tsx [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+  const headerRow = columns.map((column, cIndex) => {
+    const {title} = column;
+    return (
+      <th
+        className={style[`header-${cIndex + 1}`]}
+        key={`${title}-${cIndex + 1}`}
+        role="columnheader"
+      >
+        <div>{title}</div>
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-errors-table/index.tsx:75]: enleve la div

File: packages/@coorpacademy-components/src/molecule/expandible-errors-table/index.tsx [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+      </th>
+    );
+  });
+
+  const headerView = [
+    <th
+      className={isTableExpandible ? style[`header-0-expandible`] : style[`header-0`]}
+      key="header"
+    >
+      <div> {''}</div>
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-errors-table/index.tsx:85]: idem

File: packages/@coorpacademy-components/src/molecule/expandible-errors-table/index.tsx [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+        className={classnames(
+          isTableExpandible ? style[`col-0-expandible`] : style[`col-0`],
+          style.col
+        )}
+        key="header"
+      >
+        {some((field: string | {icon: string}) => get('icon', field) === 'invalid')(fields) ? (
+          <div className={style.upDownIcon}>
+            <ButtonLinkIconOnly
+              // eslint-disable-next-line react/jsx-no-bind
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/expandible-errors-table/index.tsx:111]: idem

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Comment by esa-coorp** [comment]
@youssefezzahi96 ton `diff coverage is 64.35%.`

File: packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.js (hunk 1 above)
Comment (esa-coorp) [reply packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.js:6]: peux tu enlever ce commentaire?

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by GhaziCoorpacademy — APPROVED** [review]

**Review by esa-coorp — APPROVED** [review]

**Pull request closed without merging**